pkg/controllers/management/node/controller.go

```diff
@@ -407,6 +394,9 @@ func (m *Lifecycle) ready(obj *v3.Node) (*v3.Node, error) {
 	}
 	defer config.Cleanup()
 
+	// Refresh first to save the compressed config, then Restore writes it to disk for machine to use
+	m.refreshNodeConfig(config, obj)
```

Comment:
Is it intended to not check for errors returned by refreshNodeConfig call?

Reply:
No, updated all calls and error message.